Packs/DemistoRESTAPI/Integrations/DemistoRESTAPI/DemistoRESTAPI.js

```diff
@@ -152,6 +154,55 @@ var deleteIncidents = function(ids_to_delete, fields_to_keep) {
     };
 };
 
+var installPackFromUrl = function(pack_url){
+    // download pack zip file
+    var res = http(
+    pack_url,
+    {
+        Method: 'GET',
+        Headers: {},
+        SaveToFile: true
+    });
+
+    if (res.StatusCode < 200 || res.StatusCode >= 300) {
+        throw 'Demisto REST APIs - Failed to download pack file from ' + pack_url;
+    }
+
+    let file_path = res.Path
+
+    // upload the pack
+    sendMultipart('contentpacks/installed/upload?skipVerify=true&skipValidation=true', file_path,'{}');
```

Comment:
`skipVerify=true&skipValidation=true` can make this command a bit risky. It's ok to skip verification/validation, but let's add these as default arguments, so it'll be better reflected to the user.

Reply:
added as arguments